item.rb

```diff
@@ -19,6 +19,11 @@ def move_to_archive()
     @archived = true
   end
 
+  def genre=(genre)
+    @genre = genre
+    genre.items.push(self) unless genre.items.include?(self)
+  end
```

Comment:
[OPTIONAL] This is not to say the method is wrong. Kindly consider adding an action word to the name of the method. For instance: `add_genre`  Doing this makes it more appropriate and meets the ruby best practices.